ui/pages/policy.js

```diff
@@ -0,0 +1,97 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import wrapPage from '../components/app-wrapper';
+import Pagers from '../components/pagers';
+import { policyData } from '../util/api/queries';
+import { Router } from '../routes';
+import PageCounter from '../components/policy/page-counter';
+import Req from '../components/policy/req';
+
+
+export class Policy extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      focusReq: props.url.query.reqId,
+    };
+  }
+
+  get reqs() {
```

Comment:
I don't think we use the class getter/setter methods anywhere else in the code base so for consistency I would change this to just be `reqs`.

FWIW, I think using the getter/setters might add some readability to the code base but it should be a consistent pattern across the component if not also the codebase